.github/workflows/performance_tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Performance Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 9 * * 0'
+
+jobs:
+  check_branch_changes:
+    runs-on: ubuntu-22.04
+    outputs:
+      changed: ${{ steps.branch_change_output.outputs.has-new-commits }}
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
`dev` now uses `checkout@v3` - please update this file when you get a chance.

Reply:
Didn't know this, thank you! Addressed in [this commit](2b4f22826184fa07d9f9b7e12146bf4f780359b1).